### ISSUE TITLE
service.getAttachTargetInfoList is not a function

### ISSUE THREAD

**Issue by nmalcolm** [facebook/nuclide#340]
Hi,

After connecting to the remote server successfully and attempting to launch the debugger (Attach to Process), I'm met with the following error in the console:

```
Uncaught (in promise) TypeError: service.getAttachTargetInfoList is not a function
```

```
pkg/nuclide/debugger/lldb-client/lib/Service.js:29
```

Code in question:

```
  const processes = [];
  await Promise.all(debuggerServices.map(async (service) => {
    const targetInfoList = await service.getAttachTargetInfoList();
    for (const targetInfo of targetInfoList) {
      processes.push(new ProcessInfo(targetUri, targetInfo));
    }
  }));
  return processes;
```

OS: Debian 8
Atom: 1.4.0 (tested with Atom 1.1.0 too)
Nuclide: 0.114.0

Unfortunately that's about as much as I can figure out. Let me know if you need any more information.

Thanks.

**Comment by nmalcolm** [comment]
If it helps, using `master` I get:

```
Uncaught (in promise) Timeout after 60000 for requestId: 30, Calling function LLDBDebuggerRpcService/getAttachTargetInfoList.
```

¯\\\_(ツ)_/¯

**Comment by nmalcolm** [comment]
Seems to be fixed by 9ecd7e30bc.
